### PR TITLE
Identify and update references new agent management section

### LINKED ISSUE #7941: Update Wazuh dashboard agents screenshots and remove VirusTotal references
# Description

In Wazuh 4.10.0 the agent views have many changes that we need to reflect in Wazuh documentation. We also integrated **VirusTotal** module into **Malware detection**, therefore we have to remove the references to VirusTotal and update any Malware Detection dashboard screenshot.

> [!note]
> For each task create a pull request to make it easier to review and integrate.

### Tasks 
- [x] #7944
- [x] #7942
- [x] #7946
- [x] #7965

### PR DESCRIPTION
**Pull request by guidomodarelli**
## Description

Identify and update all references to the new section category Agent management.

### Issue resolved
#7941

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Comment by Desvelao** [comment]
There are some unchanged references to `Endpoints Summary` that should be changed.

```
./source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:   :title: Navigating the Wazuh dashboard: Endpoints Summary
./source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:   :alt: Navigating the Wazuh dashboard: Endpoints Summary
./source/user-manual/agent/agent-management/agent-connection.rst:   :title: Wazuh dashboard Endpoints Summary menu option
./source/user-manual/agent/agent-management/agent-connection.rst:   :alt: Wazuh dashboard Endpoints Summary menu option
./source/user-manual/agent/agent-management/agent-connection.rst:   :title: Wazuh Endpoints Summary dashboard
./source/user-manual/agent/agent-management/agent-connection.rst:   :alt: Wazuh Endpoints Summary dashboard

```

**Review by Desvelao — CHANGES_REQUESTED** [review]
There are some unchanged references to `Endpoints Summary` that should be changed.

```
./source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:   :title: Navigating the Wazuh dashboard: Endpoints Summary
./source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:   :alt: Navigating the Wazuh dashboard: Endpoints Summary
./source/user-manual/agent/agent-management/agent-connection.rst:   :title: Wazuh dashboard Endpoints Summary menu option
./source/user-manual/agent/agent-management/agent-connection.rst:   :alt: Wazuh dashboard Endpoints Summary menu option
./source/user-manual/agent/agent-management/agent-connection.rst:   :title: Wazuh Endpoints Summary dashboard
./source/user-manual/agent/agent-management/agent-connection.rst:   :alt: Wazuh Endpoints Summary dashboard

```

**Comment by guidomodarelli** [comment]
> There are some unchanged references to `Endpoints Summary` that should be changed.
> 
> ```
> ./source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:   :title: Navigating the Wazuh dashboard: Endpoints Summary
> ./source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:   :alt: Navigating the Wazuh dashboard: Endpoints Summary
> ./source/user-manual/agent/agent-management/agent-connection.rst:   :title: Wazuh dashboard Endpoints Summary menu option
> ./source/user-manual/agent/agent-management/agent-connection.rst:   :alt: Wazuh dashboard Endpoints Summary menu option
> ./source/user-manual/agent/agent-management/agent-connection.rst:   :title: Wazuh Endpoints Summary dashboard
> ./source/user-manual/agent/agent-management/agent-connection.rst:   :alt: Wazuh Endpoints Summary dashboard
> ```

image

I won't change these lines because removing the word "Endpoints" means the "Summary" won't convey any information.
I think replacing "Endpoints" with "Agents" could provide more clarity.

**Review by Desvelao — DISMISSED** [review]
LGTM!

**Review by chantal-kelm — APPROVED** [review]
LGTM

File: source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst [hunk 1]
```diff
@@ -82,27 +82,27 @@ This section shows dashboards for:
    :title: Navigating the Wazuh dashboard: Cloud security
    :alt: Navigating the Wazuh dashboard: Cloud security
 
-Server management
+Agents management
 -----------------
 
-Wazuh server management offers options for managing rules, decoders, CDB lists, clusters, security configurations such as user, roles, policies and more. It also includes options for managing Wazuh agents installed on monitored endpoints.
+Wazuh agents management offers options for managing agents, agent groups, and agent configurations. It also includes options for managing Wazuh agents installed on monitored endpoints.
```
Comment (javimed) [inline source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:88]:
```suggestion
Wazuh agents management offers options for managing agents, agent groups, and agent configurations.
```

File: source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst [hunk 2]
```diff
@@ -115,6 +115,17 @@ Users can view existing groups, create new endpoint groups, and organize endpoin
    :title: Navigating the Wazuh dashboard: Endpoint Groups
    :alt: Navigating the Wazuh dashboard: Endpoint Groups
 
+Server management
+-----------------
+
+Wazuh server management offers options for managing rules, decoders, CDB lists, clusters, security configurations such as user, roles, policies and more. It also includes options for managing Wazuh agents installed on monitored endpoints.
```
Comment (javimed) [inline source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:121]:
```suggestion
Wazuh server management offers options for managing rules, decoders, CDB lists, clusters, security configurations such as user, roles, policies and more.
```

**Review by javimed — APPROVED** [review]

**Pull request merged**